tests/e2e/test_copy_project.py

```diff
@@ -395,8 +396,7 @@ def test_copy_project_e2e(kili, src_project):
         for asset_src, asset_new in zip(assets_src, assets_new):
             assert asset_src["externalId"] == asset_new["externalId"]
 
-            # Path("https://...").is_file() crashes on windows with python 3.7
-            if not asset_src["content"].startswith("http") and Path(asset_src["content"]).is_file():
+            if not asset_src["content"].startswith("http") and os.path.isfile(asset_src["content"]):
```

Comment:
we should leave the comment  to explain why we do not use `Path.is_file()`

Reply:
os.path.is_file() works well on windows, is there a reason we shouldn't use it?

Reply:
I agree, but someone might forget that there is a good reason not to use Pathlib, and will make the change the ther way around.